lib/src/view/study/study_list_screen.dart

```diff
@@ -0,0 +1,472 @@
+import 'package:lichess_mobile/src/widgets/user_full_name.dart';
+import 'package:logging/logging.dart';
+import 'package:timeago/timeago.dart' as timeago;
+
+final _logger = Logger('StudyListScreen');
+
+String studyCategoryL10n(StudyCategory category, BuildContext context) =>
+    switch (category) {
+      StudyCategory.all => context.l10n.studyAllStudies,
+      StudyCategory.mine => context.l10n.studyMyStudies,
+      StudyCategory.member => context.l10n.studyStudiesIContributeTo,
+      StudyCategory.public => context.l10n.studyMyPublicStudies,
+      StudyCategory.private => context.l10n.studyMyPrivateStudies,
+      StudyCategory.likes => context.l10n.studyMyFavoriteStudies,
+    };
+
+// TODO l10n
+String studyListOrderL10n(StudyListOrder order, BuildContext context) =>
+    switch (order) {
+      StudyListOrder.hot => context.l10n.studyHot,
+      StudyListOrder.newest => context.l10n.studyDateAddedNewest,
+      StudyListOrder.oldest => context.l10n.studyDateAddedOldest,
+      StudyListOrder.updated => context.l10n.studyRecentlyUpdated,
+      StudyListOrder.popular => context.l10n.studyMostPopular,
+    };
+
+/// A screen that displays a paginated list of studies
+class StudyListScreen extends ConsumerWidget {
+  const StudyListScreen({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final isLoggedIn = ref.watch(authSessionProvider)?.user.id != null;
+
+    final filter = ref.watch(studyFilterProvider);
+    final categorySection =
+        isLoggedIn ? ' • ${studyCategoryL10n(filter.category, context)}' : '';
+    final title = Text(
+      '${context.l10n.studyMenu}$categorySection • ${studyListOrderL10n(filter.order, context)}',
+    );
+
+    return PlatformScaffold(
+      appBar: PlatformAppBar(
+        title: title,
+        actions: [
+          AppBarIconButton(
+            icon: const Icon(Icons.tune),
+            semanticsLabel: 'Filter studies',
+            onPressed: () => showAdaptiveBottomSheet<void>(
+              context: context,
+              builder: (_) => _StudyFilterSheet(
+                isLoggedIn: isLoggedIn,
+              ),
+            ),
+          ),
+        ],
+      ),
+      body: SafeArea(child: _Body(filter: filter)),
+    );
+  }
+}
+
+class _StudyFilterSheet extends ConsumerWidget {
+  const _StudyFilterSheet({required this.isLoggedIn});
+
+  final bool isLoggedIn;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final filter = ref.watch(studyFilterProvider);
+
+    return SafeArea(
+      child: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            const SizedBox(height: 12.0),
+            // If we're not logged in, the only category available is "All"
+            if (isLoggedIn) ...[
+              Filter<StudyCategory>(
+                // TODO mobile l10n
```

Comment:
This one (and also "Sort by") do not exist in the Web UI and need will mobile-specific translations at some point in the future